dev/com.ibm.ws.grpc_fat/fat/src/com/ibm/ws/fat/grpc/HelloWorldCDITests.java

```diff
@@ -84,6 +84,10 @@ public void preTest() {
 
     @After
     public void afterTest() {
+        // Setting serverConfigurationFile to null forces a server.xml update (when GrpcTestUtils.setServerConfiguration() is first called) on the repeat run
+        // If not set to null, test failures may occur (since the incorrect server.xml could be used)
+        serverConfigurationFile = null;
```

Comment:
This class is skip for EE9 run, so you could remove it. 
Otherwise, this should be moved to @AfterClass instead.

Reply:
good catch. I'll move it to the @AfterClass so we don't have to worry about it in the future